templates/success.html

```diff
@@ -14,14 +15,16 @@
     <h2>You are registered for the DESC August 2022 collaboration meeting!</h2>
     <br>
     <br>
-    {{data.first_name}}, we are looking forward to seeing you at our virtual meeting!<br>
+    {{data.first_name}}, we are looking forward to seeing you at the meeting!<br>
 
     <br>
-    The <a href="https://confluence.slac.stanford.edu/x/fQzUEw" target="_blank">Participants tab</a> on the confluence page links to a live participant list, where you can confirm your
+    The <a href="https://confluence.slac.stanford.edu/x/fQzUEw" target="_blank">Participants tab</a> on the confluence
+    page links to a live participant list, where you can confirm your
     registration.
     <br>
     <br>
-    And have a look at the <a href="https://confluence.slac.stanford.edu/x/DwzUEw" target="_blank">agenda page</a> to check the program of our virtual meeting!
+    And have a look at the <a href="https://confluence.slac.stanford.edu/x/DwzUEw" target="_blank">agenda page</a> to
```

Comment:
Link should be changed to 
https://confluence.slac.stanford.edu/x/ooXDF
(though page has nothing useful in it yet)